documentation20/en/11.administrator/docs.md

```diff
@@ -406,6 +406,51 @@ After TDengine is started, it will automatically create a monitoring database lo
 
 The collection of these monitoring metrics is turned on by default, but you can modify option monitor in the configuration file to turn it off or on.
 
+### TDinsight - Monitor TDengine with Grafana + Data Source
+
+Starting from v2.3.3.0, TDengine `log` database provides more metrics for resources and status monitoring. Here we are introducing a zero-dependency monitoring solution - we call it TDinsight - with only Grafana. The full documentations are located in [GitHub](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.md).
+
+For a shortcut, we provide a automation shell script [`TDinsight.sh`](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.sh) to help setup TDinsight on Grafana server.
```

Comment:
Suggestion:
We provide an automation shell script [`TDinsight.sh`](https://github.com/taosdata/grafanaplugin/blob/master/dashboards/TDinsight.sh) as a shortcut to help setup TDinsight on the Grafana server.